titus-client/src/main/java/com/netflix/titus/runtime/endpoint/v3/grpc/GrpcObjectsCacheConfiguration.java

```diff
@@ -0,0 +1,38 @@
+package com.netflix.titus.runtime.endpoint.v3.grpc;
+
+import com.netflix.archaius.api.annotations.Configuration;
+import com.netflix.archaius.api.annotations.DefaultValue;
+
+@Configuration(prefix = "titus.grpc.cache")
+public interface GrpcObjectsCacheConfiguration {
+
+    /**
+     * @return whether or not the cache is enabled
+     */
+    @DefaultValue("true")
+    boolean isGrpcObjectsCacheEnabled();
```

Comment:
Should we separate into two properties for jobs and tasks in case we want to disable caching for either one for performance reasons?

Reply:
I think we should be able to disable cache as a whole in case there is a problem with it. It is very unlikely we might get odd behavior with one entity, and normal with another.